<?xml version="1.0" ?>
<launch>
  <arg name="pose_file" default="overhead_rob" />
  <arg name="load_board" default="true" />
  <arg name="start_srv" default="true" />

  <group if="$(arg start_srv)">
    <include file="$(find excel_calibration)/launch/calib_server.launch">
      <arg name="delete_job" value="true" />
    </include>
  </group> 

  <node pkg="excel_calibration" type="auto_calib_node" name="auto_calib_node" 
        output="screen">
    <rosparam command="load" 
              file="$(find excel_calibration)/joint_pose_data/$(arg pose_file)_poses.yaml" 
              param="joint_poses" />
    <param name="sim" value="false" />
    <param name="load_board" value="$(arg load_board)" />
  </node>
</launch>
